tests/Properties.hs

```diff
@@ -2333,7 +2340,10 @@ bb_tests =
     , testProperty "zipWith"        prop_zipWithBB
     , testProperty "zipWith"        prop_zipWithCC
     , testProperty "zipWith"        prop_zipWithLC
---  , testProperty "zipWith'"       prop_zipWith'BB
+    , testProperty "packZipWith"    prop_packZipWithBB
+    , testProperty "packZipWith"    prop_packZipWithLL
+    , testProperty "packZipWith"    prop_packZipWithBC
+    , testProperty "packZipWith"    prop_packZipWithLC
```

Comment:
(Our test suite is a terrible mess, we really need to switch to a framework, which supports test groups.) 
In the meantime could you please make test names for `zipWith` and `packZipWith` distinct? Even something like `zipWithBB`, `zipWithCC`, `packZipWithLC`, etc. will do.

Reply:
Done